src/custom/components/Header/HeaderMod.tsx

```diff
@@ -1,9 +1,10 @@
 import { ChainId } from '@uniswap/sdk'
 import React, { useState } from 'react'
 import { Text } from 'rebass'
-// import { NavLink } from 'react-router-dom'
-// import { darken } from 'polished'
+import { NavLink } from 'react-router-dom'
+import { darken } from 'polished'
 // import { useTranslation } from 'react-i18next'
+import { LinkType } from './index'
```

Comment:
Note: Modifies HeaderMod.tsx (because of needing the LinkType for the passed HeaderLinks)